src/GenAPI/Microsoft.DotNet.GenAPI.Task/Microsoft.DotNet.GenAPI.Task.csproj

```diff
@@ -10,6 +10,8 @@
     <NoWarn>$(NoWarn);NU5128</NoWarn>
     <Nullable>enable</Nullable>
     <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);_AddBuildOutputToPackageCore;_AddBuildOutputToPackageDesktop</TargetsForTfmSpecificContentInPackage>
+    <_MicrosoftCodeAnalysisVersion>4.5.0-3.22601.3</_MicrosoftCodeAnalysisVersion>
+    <_MicrosoftCodeAnalysisVersion Condition="'$(DotNetBuildFromSource)' == 'true'">$(MicrosoftCodeAnalysisPackageVersion)</_MicrosoftCodeAnalysisVersion>
```

Comment:
These two properties are dead code in this project file as the PackageReferences to Microsoft.CodeAnalysis aren't defined here anymore.

This should instead now be changed here: https://github.com/dotnet/sdk/blob/bf4da7e01882fe5b680eaa9d30d6f33fbdf1f23d/src/GenAPI/Microsoft.DotNet.GenAPI/Microsoft.DotNet.GenAPI.csproj#L9-L10

Reply:
updated. thanks